modules/generators.py

```diff
@@ -69,7 +69,7 @@ async def _claude_legacy_factory(
         wrapped_messages.append(
             {
                 "role": role,
-                "content": content,
+                "content": content,  # type: ignore[reportArgumentType]
```

Comment:
Pyright was complaining about the fact that content can be both string and dict[str, Any]. I tried to properly define the types to resolve the complaint but it's complex and added a lot of peripheral code so I decided to just ignore it for now.